### PR TITLE
Test to validate .vox files

### LINKED ISSUE #62: .vox file validation test
we need a test that simply loads all voxel files in the voxygen/vox files and tries to load them into a model.
This way Voxel Artists can know if there is any incompatibility between their uploads and the engine.

The method already exists, you just need to reuse the functions from voxygen/src/game.rs for all those files. and maybe write a test in .gitlab-ci.yaml to automatic run this

### PR DESCRIPTION
**Pull request by treenix**
This fixes #62. No changes needed in the current travis config, [I tested it on my fork](https://travis-ci.org/treenix/game/builds/399914862).

### CODE REVIEW

File: voxygen/tests/validate_vox_files.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+#[test]
+fn validate_vox_files() {
+    let paths = fs::read_dir("./vox").unwrap();
+    
+    for path in paths {
+        let path_string = path.unwrap().path().into_os_string().into_string().unwrap();
+        let vox = dot_vox::load(&path_string);
+        assert_eq!(true, vox.is_ok(), "Failed to validate file '{:?}'", path_string);
+    }
```
Comment (xMAC94x) [inline voxygen/tests/validate_vox_files.rs]: can you put a assert here that at least 1 file is checked in case the folder changes or so?

Comment (xMAC94x) [reply voxygen/tests/validate_vox_files.rs]: on the other hand this should already be checked by the unwrap in line 6 or?

Comment (treenix) [reply voxygen/tests/validate_vox_files.rs]: Yes, it is already checked there

Comment (treenix) [reply voxygen/tests/validate_vox_files.rs]: Would it make sense to test the folder not being empty?

Comment (xMAC94x) [reply voxygen/tests/validate_vox_files.rs]: yeah might be, can you adjust the test ?

**Comment by treenix** [comment]
@xMAC94x The test fails now if the folder is empty

**Comment by zesterer** [comment]
All good to merge @xMAC94x?

**Pull request merged**